lib/reverb/Reverb.cc

```diff
@@ -438,7 +438,7 @@ void MixxxPlateX2::processBuffer(const sample_t* in, sample_t* out, const uint f
 	// set bandwidth
 	input.bandwidth.set(exp(-M_PI * (1. - (.005 + .994*bandwidthParam))));
 	// set decay
-	sample_t decay = .749*decayParam;
+	sample_t decay = .890*decayParam;
```

Comment:
This function has a lot of magic numbers. Does anyone know where they came from? Guessing and checking until it sounded good? @rryan do you know?

Reply:
This PR is just changing the max decay paramter until I found a value I liked. Now I find the reverb way more useful: you can generate a long reverb tail and filter it out with the moog filter. Sweet :)